cms/djangoapps/contentstore/views/organization.py

```diff
@@ -18,6 +20,12 @@ class OrganizationListView(View):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         """Returns organization list as json."""
-        organizations = get_organizations()
-        org_names_list = [(org["short_name"]) for org in organizations]
+
+        if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS):
+            organizations = get_organizations()
+            org_names_list = [(org["short_name"]) for org in organizations]
```

Comment:
nit: use single quotes in code.
double quotes are used in docstrings or in comments.